<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HandoverDlg</class>
 <widget class="QDialog" name="HandoverDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>271</width>
    <height>187</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>交班</string>
  </property>
  <widget class="QGroupBox" name="staticGp">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>271</width>
     <height>95</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(34, 173, 56);</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>5</y>
      <width>120</width>
      <height>85</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <widget class="QLabel" name="labStaticTradeVal">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;Arial&quot;;
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>0.00 / 元</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="labStatic">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>4</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: bold 12pt &quot;Arial&quot;;
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>今日交易</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="labStaticTradeTms">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>60</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;Arial&quot;;
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>0.00 / 笔</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>5</y>
      <width>120</width>
      <height>85</height>
     </rect>
    </property>
    <widget class="QLabel" name="labStaticRefundVal">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;Arial&quot;;
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>0.00 / 元</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="labStatic_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>4</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: bold 12pt &quot;Arial&quot;;
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>今日退款</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="labStaticRefundTms">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>60</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;Arial&quot;;
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>0.00 / 笔</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_6" native="true">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>18</y>
      <width>2</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pbtOK">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: bold 10pt &quot;Arial&quot;;</string>
   </property>
   <property name="text">
    <string>确定（Enter）</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbtCancel">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>150</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: bold 10pt &quot;Arial&quot;;</string>
   </property>
   <property name="text">
    <string>取消（ESC）</string>
   </property>
  </widget>
  <widget class="QLabel" name="labName">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>251</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>username</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
